tests/unit/Guzzle/ClientFactoryTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Mediawiki\Api\Test\Unit\Guzzle;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\HandlerStack;
+use Mediawiki\Api\Guzzle\ClientFactory;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * @author Addshore
+ *
+ * @todo test interaction with logger
```

Comment:
Again I think this comment was probably copied over in error, there isn't much interaction with the logger class in here to test!